params/config.go

```diff
@@ -28,7 +28,7 @@ var (
 	ColosseumGenesisHash = common.HexToHash("0x07107d5523286f6e9bda586242abaf77a262a6763b63969db7a3635679ce6b63")
 	GardenGenesisHash    = common.HexToHash("0xe8c63ec6e47b9f9c126c505eaddc639165a3c952938f791705cb0ef4a7e3b184")
 	OrchardGenesisHash   = common.HexToHash("0x23962df067cb177b33fcf58a3fd6e07a1cde9d2d856e944af82ef706ef4c7bf2")
-	LocalGenesisHash     = common.HexToHash("0xe2a3d48409a943a0755d94e4b4e1edca45b36e86d4d46f6a7b01c60684e0aabf")
+	LocalGenesisHash     = common.HexToHash("0x4392348c52ff19af0dc7215b65b89f25cf4e0812cc6f10e79b5ccdf697067028")
```

Comment:
Why did the local genesis hash change in this commit, but not the others?

Reply:
lBlockchain technology is developing rapidly and new projects are starting to appear in the system. Quai Network aims to be the fastest and most secure blockchain in the crypto universe. It is trying to provide a decentralized solution that can be scaled and integrated according to global trade.